src/components/Artifacts/Artifact.vue

```diff
@@ -20,10 +20,19 @@ export default {
 
 <template>
   <div
+    v-if="artifact.kind == 'md'"
     class="artifact md grey--text text--darken-3 mx-4"
-    v-html="mdParser(artifact.data.markdown || artifact.data.link)"
+    v-html="mdParser(artifact.data.markdown)"
   >
   </div>
+  <div v-else-if="artifact.kind == 'link'">
+    <router-link :to="{ path: artifact.data.link }">
```

Comment:
Is this going to handle external links? [This artifact external link](https://deploy-preview-559--prefect-ui.netlify.app/prefect/flow-run/5aac851f-47af-4658-b25c-71b487a811b1?artifacts=) now goes to the 404 page. We could maybe add a check for https to decide how to handle a link?

Reply:
Ah you know what, I didn't realize this could be used to generate external links - will update with a check for that.

Reply:
Fixed!